plugin/trino-exchange/src/test/java/io/trino/plugin/exchange/containers/MinioStorage.java

```diff
@@ -89,6 +89,7 @@ public void close()
                 .put("exchange.base-directory", "s3n://" + minioStorage.getBucketName())
                 // TODO: enable exchange encryption after https is supported for Trino MinIO
                 .put("exchange.encryption-enabled", "false")
+                .put("exchange.sink-target-file-size", "10MB")
```

Comment:
Could you please make sure there's at least a single test that triggers this condition (creates more than a single file per partition)?

Reply:
https://github.com/trinodb/trino/blob/9c3961712ca813dbac303a239ac98921d5e27489/plugin/trino-exchange/src/test/java/io/trino/plugin/exchange/AbstractTestExchangeManager.java#L168 exercises the split code path.